event_routing_backends/processors/xapi/event_transformers/navigation_events.py

```diff
@@ -136,12 +138,34 @@ def get_context(self):
             `Context`
         """
         context = Context(
-            registration=get_anonymous_user_id(
-                self.extract_username_or_userid()
-            )
+            contextActivities=self.get_context_activities()
         )
         return context
 
+    def get_context_activities(self):
```

Comment:
@rehanedly I think we should move this method in the `XApiTransformer` to make available to all transformers and use it where possible instead of overriding it in child transformers.

Reply:
@ziafazal I have fixed it